Sources/App/Core/Plausible.swift

```diff
@@ -31,16 +31,9 @@ enum Plausible {
         case badge = "/api/packages/{owner}/{repository}/badge"
         case package = "/api/packages/{owner}/{repository}"
         case packageCollections = "/api/package-collections"
+        case rss = "/rss"
         case search = "/api/search"
-    }
-
-    enum Identifier {
-        case anonymous
-        case identifier(String)
-
-        var props: [String: String] {
-            fatalError("fix me")
-        }
+        case sitemap = "/sitemap"
```

Comment:
These (`/rss` and `/sitemap`) aren't real paths, they're just for filtering in Plausible. I've just picked something that seemed sensible.

Reply:
Yes this is fine. No need for the `.xml`.